pkg/common/common.go

```diff
@@ -106,3 +106,33 @@ func NewLogLevel(lvl string) (zapcore.LevelEnabler, bool) {
 		return zapcore.DebugLevel, true
 	}
 }
+
+func CustomLogger() *zap.Logger {
+
+	logLvl, isDevelopment := NewLogLevel(util.GetEnvWithDefault("LOG_LEVEL", "Debug"))
+
+	conf := zap.Config{
+		Level:            zap.NewAtomicLevelAt(logLvl),
+		Development:      isDevelopment,
+		Encoding:         "json",
```

Comment:
Should the encoding be comfigurable via environment variable? Valid values are currently `json` or `console`.

https://github.com/uber-go/zap/blob/v1.21.0/config.go#L75-L78